pkgs/dartpad_ui/web/styles/index.css

```diff
@@ -0,0 +1,112 @@
+.CodeMirror {
+  height: 100%;
+}
+
+.squiggle-info {
+  border-bottom: 2px solid #7ea7d8;
+}
+
+.squiggle-warning {
+  border-bottom: 2px solid #d9d077;
+}
+
+.squiggle-error {
+  border-bottom: 2px solid #ef5350;
+}
+
+.squiggle-deprecation {
+  text-decoration: line-through;
+  text-decoration-thickness: 10%;
+}
+
+.CodeMirror-simplescroll-horizontal div,
+.CodeMirror-simplescroll-vertical div {
+  position: absolute;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+.CodeMirror-simplescroll-horizontal,
+.CodeMirror-simplescroll-vertical {
+  position: absolute;
+  z-index: 6;
+}
+
+.CodeMirror-simplescroll-horizontal {
+  bottom: 0;
+  left: 0;
+  height: 9px;
+}
+
+.CodeMirror-simplescroll-horizontal div {
+  bottom: 0;
+  height: 100%;
+}
+
+.CodeMirror-simplescroll-vertical {
+  right: 0;
+  top: 0;
+  width: 9px;
+}
+
+.CodeMirror-simplescroll-vertical div {
+  right: 0;
+  width: 100%;
+}
+
+#splash {
+  position: fixed;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 999;
+  transition: opacity 500ms;
+}
+
+#splash.light {
+  background-color: #f5f5f7;
```

Comment:
I assume this color is the color we'd settle into after we've started up in the light theme?

Reply:
Yeah, that's right.